File: Command/Javascript/Editor.js
class Editor {
  constructor() {
    this.commandList = [];
  }

  input(command) {
    this.commandList.push(command);
  }

  executeCommands() {
    this.commandList.forEach(command => {
      command.execute();
    });

    this.commandList = [];
  }
}

export default Editor;

File: Command/Python 3/Editor.py
class Editor:
  def __init__(self):
    self.__commandList = []
  
  def input(self, command):
    self.__commandList.append(command)
  
  def executeCommands(self):
    for command in self.__commandList:
      command.execute()
    
    self.__commandList = []
